build.sh

```diff
@@ -16,5 +27,5 @@ mkdir -p $P/scratch;
 cd $P/scratch;
 rm -rf $P/install;
 $SRC/configure --prefix=$P/install --with-mountutildir=$P/install/sbin --with-initdir=$P/install/etc --localstatedir=/var --enable-bd-xlator=yes --enable-debug --silent
-make install CFLAGS="-g -O0 -Wall -Werror" -j 4 >/dev/null
+make install CFLAGS="-g -O0 -Wall -Werror" -j ${nproc} >/dev/null
```

Comment:
@kshlm @jdarcy @raghavendra-talur 

After my investigation on make jobs this is the summary:

 "make -j" where if you don't give it a parameter it uses a crazy default (basically, run as many as jobs as it can in parallel - even a hundred).

Actually, number of threads should be proportional to the hardware threads (may be 1.5x),  'make -j' can't limit to hardware_threads X nproc [ X 1.5 ] and goes far beyond the limit.

In fact I have seen VM hung followed by OOM kill in my local VM while compiling linux kernel with 'make -j'.

Basically In the VMs "Thread(s) per core:   1" (mostly), hence its a best practice to limit it to nprocs meaning use 'make -j${nproc}'.

So I still think this patch is valid  :)